packages/legacy-workbench/src/js/angular/guides/steps/complex/ttyg/ask-ttyg-agent/plugin.js

```diff
@@ -35,11 +33,111 @@ PluginRegistry.add('guide.step', [
                         },
                         hide: () => () => {
                             const elementSelector = GuideUtils.getGuideElementSelector('contenteditable');
-                            // Remove the "keydown" listener of element. It is important when step is hidden.
+                            // Remove the "keydown" listener of the element. It is important when the step is hidden.
                             $(elementSelector).off('keydown');
                         },
                         disableNextFlow: true
                     }, options)
```

Comment:
The options must come before the `show`, `hide`, `url`, and `elementSelector` options so that they cannot be overridden. These options are critical for this step, and if they are overridden, the step will stop working properly.